Readme.md

```diff
@@ -148,6 +166,40 @@ The current lead maintainer is [Douglas Christopher Wilson](https://github.com/d
 
 [List of all contributors](https://github.com/expressjs/express/graphs/contributors)
 
+### Technical Committee
+
+* [UlisesGascon](https://github.com/UlisesGascon) - **Ulises Gascon** (he/him)
+* [jonchurch](https://github.com/jonchurch) - **Jon Church**
+* [wesleytodd](https://github.com/wesleytodd) - **Wes Todd**
+* [LinusU](https://github.com/LinusU) - **Linus Unnebäck**
+* [blakeembrey](https://github.com/blakeembrey) - **Blake Embrey**
+* [sheplu](https://github.com/sheplu) - **Jean Burellier**
+* [crandmck](https://github.com/crandmck) - **Rand McKinney**
+
+### Triagers
+
+* [aravindvnair99](https://github.com/aravindvnair99) - **Aravind Nair**
+* [carpasse](https://github.com/carpasse) - **Carlos Serrano**
+* [CBID2](https://github.com/CBID2) - **Christine Belzie**
+* [enyoghasim](https://github.com/enyoghasim) - **David Enyoghasim**
+* [UlisesGascon](https://github.com/UlisesGascon) - **Ulises Gascon**
+* [mertcanaltin](https://github.com/mertcanaltin) - **Mert Can Altin**
+* [0ss](https://github.com/0ss) - **Salah**
+* [import-brain](https://github.com/import-brain) - **Eric Cheng** (he/him)
+* [3imed-jaberi](https://github.com/3imed-jaberi) - **Imed Jaberi**
+* [dakshkhetan](https://github.com/dakshkhetan) - **Daksh Khetan** (he/him)
+* [lucasraziel](https://github.com/lucasraziel) - **Lucas Soares Do Rego**
+* [Sushmeet](https://github.com/Sushmeet) - **Sushmeet Sunger**
+
+## Emeritus Team Members (Click to expand)
+<details>
+  <summary>Emeritus Team Members</summary>
+  * List out Emeriti here
+    * Name
+    * Contribution (optional)
+</details>
```

Comment:
I like this addition from @CBID2, but I have two suggestions.

We should try to maintain the same format as the "non emeritus" and follow the heading semantics, like:

```markdown
<details>
  <summary>Triage emeriti members</summary>
  * [UlisesGascon](https://github.com/UlisesGascon) - **Ulises Gascon**
</details>
```

@mertcanaltin, if we want to release these changes now, we can simply remove this reference and create a separate PR for the emeritus team members. Otherwise, we can include the emeritus team additions in this PR.

Reply:
@mertcanaltin I was checking last changes, thanks for include the emeritus!

let me share also a visual comparative of what motivates my change:

**Express proposal (35ef8e5d58e03c1da97fef221f3ca1d3cff2e991)**
![Screenshot from 2024-05-01 11-43-01](https://github.com/expressjs/express/assets/5110813/a3d95f71-ecb4-4dcc-b425-55c16fb3bf19)




_[ref](https://github.com/expressjs/express/blob/35ef8e5d58e03c1da97fef221f3ca1d3cff2e991/Readme.md)_

**Node.js currently (Collapsed)**
![Screenshot from 2024-05-01 11-43-50](https://github.com/expressjs/express/assets/5110813/5e84b352-e84d-468d-9965-823d38e2bf6b)

_[ref](https://github.com/nodejs/node/blob/main/README.md)_


**Node.js currently (Opened)**


![Screenshot from 2024-05-01 11-43-45](https://github.com/expressjs/express/assets/5110813/d2bd0cbb-8545-4f03-b96e-eaaa89e97c0e)



_[ref](https://github.com/nodejs/node/blob/main/README.md)_

Reply:
thank you very much, I think it would be correct to add the names as in node js, I will send another update

I wonder if you have a suggestion 🙏🚀

Reply:
I sent a commit that organizes 🚀